Presentation.Web/app/components/global-admin/global-admin-misc.view.html

```diff
@@ -14,8 +14,43 @@ <h3>Rapport over brudte links</h3>
         <p><strong>Antal registrerede fejl:</strong> {{brokenLinksVm.brokenLinksCount}}</p>
         <a class="btn btn-default" id="brokenLinksReportAnchor" href="{{brokenLinksVm.linkToCurrentReport}}">Hent rapporten</a>
     </div>
-    <div class="well" ng-if="!brokenLinksVm.available">
-        <p>Rapporten over brudte links er endnu ikke tilgængelig. Forventet tilgængelighed er "Mandag i næste uge".</p>
+     <div class="well" ng-if="!brokenLinksVm.available">
+         <p>Rapporten over brudte links er endnu ikke tilgængelig. Forventet tilgængelighed er "Mandag i næste uge".</p>
+     </div>
+
+    <!--TODO: Change text-->
+    <h3>*INSERT TEXT IN DANISH*</h3>
+    <div class="well">
+        <div class="row">
+            <input id="add-new-user-for-delete"
+                   class="col-xs-11"
+                   ui-select2="::userOptions"
+                   placeholder="Vælg bruger"
+                   ng-model="selectedUser">
+            <button class="btn btn-danger btn-sm col-xs-1"
+                    ng-if="userSelected"
+                    data-confirmed-click="removeUser(selectedUser.id)"
+                    data-confirm-click="Er du sikker på at du vil slette?">
+                <!--TODO: Change text-->
+                Delete
+            </button>
+        </div>
+        <table class="full-width table table-striped"
```

Comment:
missing `ng-if="userOrganizations && userOrganizations.length > 0"`
Also wrap it in a div and let that one own the ng-if.
Before the header, add the text:

Brugeren er stadig medlem af følgende organisationer